WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpViewModel.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import Yosemite
+
+class InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpViewModel: ObservableObject {
+    let completion: () -> ()
```

Comment:
Nit/question: We seem to use both `() -> () `and `() -> Void` indistinctively across the codebase, is there a guideline or preference? The former seems to be used much less than the later.

Reply:
There's no difference between them... and I've never thought too hard about it, used both instinctively where they felt right.

[Erica Sadun settled on `() -> Void` early on](https://ericasadun.com/2015/05/11/swift-vs-void/), and so did Apple's docs teams. As you say, we already use that form way more frequently too. Who am I to argue!